torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -5,7 +5,18 @@
 from torch.utils._python_dispatch import TorchDispatchMode
 
 
-funcol = torch.ops.c10d_functional
+native = torch.ops._c10d_functional
+legacy = torch.ops.c10d_functional
```

Comment:
We may want to be even more verbose on these names as there were other parts of funcol where we refer to c10d collectives as legacy.

How about funcol_py and funcol_native